02-express-tutorial/answers/14-router-auth.js

```diff
@@ -0,0 +1,13 @@
+const express = require('express')
+const router = express.Router()
+
+router.post('/', (req, res) => {
+  const { name } = req.body
+  if (name) {
+    return res.status(200).send(`Welcome ${name}`)
+  }
+
+  res.status(401).send('Please Provide Credentials')
+})
+
+module.exports = router
```

Comment:
Since you have this file under `routes/auth.js` it doesn't need to be copied here as well.

Reply:
I commented out this code. Or were you suggesting that I delete this exercise file?